crates/protocol/genesis/src/superchain/chain_list.rs

```diff
@@ -62,11 +62,21 @@ pub struct Chain {
     pub explorers: Vec<String>,
     /// The Superchain Level.
     pub superchain_level: u64,
+    /// Governed by Optimism flag.
+    #[cfg_attr(feature = "tabled", tabled(skip))]
```

Comment:
The added fields in this struct are already in the superchain-registry configs in the etc directory. I updated the struct so that parsing the superchain-registry configs do not strip them out.